src/components/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -243,14 +243,14 @@ $fullscreen-modal-padding: 64px;
   width: 100vw;
   z-index: 9999;
 
-  @media only screen and (min-width: ($viewport-md + 1px)) {
+  @media (min-width: ($viewport-md)) {
```

Comment:
nit: you don't need the parenthesis around the variable anymore 😬

Reply:
Fixed